src/index.html

```diff
@@ -11,10 +11,24 @@
             <h1>News Portal</h1>
         </header>
         <main>
+            <div class="filter"></div>
             <div class="sources buttons"></div>
             <div class="news"></div>
         </main>
         <footer>
+            <div class="footer__container">
+                <div class="container__git">
+                    <span class="footer__text">©</span>
+                    <span class="footer__text">2022</span>
+                    <a class="footer__link" href="https://github.com/RigelTLT">
+              Github
+            </a>
```

Comment:
Pay more attention to code formatting, it's very important. 
Well-formatted code is not only nice to look at, but it makes it easier to understand and find errors.